src/static/riot/datasets/management.tag

```diff
@@ -214,7 +226,10 @@
             self.update_datasets()
         })
 
-        self.show_info_modal = function (row) {
+        self.show_info_modal = function (row, e) {
+            if (e.target.type === 'checkbox') {
+                return
```

Comment:
nice! Can you add a comment about why we did this? although it is probably pretty clear, may not be in 2 years